### PR TITLE
Adding the Breeze agent to the node version

### PR DESCRIPTION
**Pull request by dangershony**
I am considering we should change the default agent from `StratisBitcoin` to be `StratisNode`.  

Alternatively we can also enforce a certain structure in the agent name to `stratis:[node-description]:version`.
Then for the fullnode it will be `stratis:fullnode:1.0.2` and for breeze it will be `stratis:breeze:1.0.2`

We could also put macros to for example for the version, Breeze has its own version this will allow to override version.

### CODE REVIEW

File: Stratis.BreezeD/Program.cs [hunk 1]
```diff
@@ -26,6 +26,7 @@ public static void Main(string[] args)
             var apiUri = args.GetValueOf("apiuri");
             var isTestNet = args.Contains("-testnet");
             var isStratis = args.Contains("stratis");
+            var agent = "Breeze(spv)";
```
Comment (bokobza) [inline Stratis.BreezeD/Program.cs]: I think the agent should be "Breeze", no need to specify spv.

File: Stratis.Bitcoin/Connection/ConnectionManager.cs [hunk 2]
```diff
@@ -75,8 +75,8 @@ public ConnectionManager(Network network, NodeConnectionParameters parameters, N
             this.logger = loggerFactory.CreateLogger(this.GetType().FullName);
         }
         public void Start()
-		{
-			this.parameters.UserAgent = "StratisBitcoin:" + GetVersion();
+        {
+            this.parameters.UserAgent = $"{this.NodeSettings.Agent}:{this.GetVersion()}";
```
Comment (bokobza) [inline Stratis.Bitcoin/Connection/ConnectionManager.cs]: please review formatting here.

File: Stratis.BreezeD/Program.cs (hunk 1 above)
Comment (dangershony) [reply Stratis.BreezeD/Program.cs]: I added spv so observers in the network that dont know what breeze is will get an idea of the nodes purpose. (It applies mostly for the bitcoin netwrok).

Comment (dangershony) [reply Stratis.BreezeD/Program.cs]: Ok I will change it for now and remove the `spv`

**Comment by bokobza** [comment]
totah

**Pull request merged**